buildmaster/LHCB_DY_13TEV_DIELECTRON/metadata.yaml

```diff
@@ -0,0 +1,53 @@
+# Previously called `LHCB_Z_13TEV_DIELECTRON`
+# author: "Tanjona R. Rabemananjara"
+setname: "LHCB_DY_13TEV_DIELECTRON"
+
+version: 1
+version_comment: "First implementation"
+
+# References
+arXiv:
+  url: "https://arxiv.org/abs/1607.06495"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1477581"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1477581"
+  version: 1
+
+nnpdf_metadata:
+  nnpdf31_process: "DY NC"
+  experiment: "LHCb"
+
+implemented_observables:
+- observable_name: "y"
+  observable:
+    description: "Differential cross-section of Z-->ee as a function of Z-rapidity"
+    label: r"$d\isgma / d|y|$"
+    units: "[fb]"
+  process_type: "DY"
+  ndata: 17
```

Comment:
`ndata` is no longer necessary, we read it directly from the data.

Reply:
`\isgma` should be `\sigma`.